test/functional/fixtures/es_archiver/discover/data.json

```diff
@@ -41,3 +55,31 @@
     }
   }
 }
+
+{
+  "type": "doc",
+  "value": {
+    "id": "0",
+    "index": "discover-routing-test",
+    "routing": "0",
+    "source": {
+      "@timestamp": "2015-09-20T08:00:00.000Z",
+      "name": "a",
+      "value": 500
+    }
+  }
+}
+
+{
+  "type": "doc",
+  "value": {
+    "id": "1",
+    "index": "discover-routing-test",
+    "routing": "1",
```

Comment:
Is this something that we can detect from ES and store with the document when we save es archives? It's not uncommon for people to update esArchives by loading them into ES and then saving them, but this only works if the esArchiver can fully recreate the archive data from ES.

Reply:
Yes, it is. This is part of the metadata returned on the search response, and it's why I actually needed to add examples here.

Reply:
Mind adding it to the records here then? https://github.com/wylieconlon/kibana/blob/discover-context-performance/packages/kbn-es-archiver/src/lib/docs/generate_doc_records_stream.ts#L55-L65